action.yml

```
@@ -68,6 +68,22 @@ inputs:
   extra-parameters:
     description: Additional parameters added to the secure-inline-scan container execution.
     required: false
+  mode:
+    description: 'Mode of operation. Can be "vm" or "iac".'
+    required: false
+    default: image
```

Shouldn't it be set to either `vm` or `iac`?
```suggestion
    default: vm
```